joomla/joomla-cms
# Show location of cause of deprecation messages

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/38137 .

### Summary of Changes
Add file and line reference to deprecation exception messages.


### Testing Instructions
Enable deprecated method calls logging (Admin panel -> global configuration -> logging)
Run just about any view in the back end.
View file /administrator/logs/deprecated.php to see reports of deprecated calls.


### Actual result BEFORE applying this Pull Request
Deprecation messages appear without any reference to location of the piece code causing the exception.


### Expected result AFTER applying this Pull Request
Deprecation messages contain a full file name and line number locating the cause of the exception.
Documentation Changes Required


### Documentation Changes Required
None
